assets/css/user.css

```diff
@@ -0,0 +1,132 @@
+/*
+ * User pages CSS
+ *
+ * Used on all authenticated pages
+ * (subscriptions, account management, etc...)
+ *
+ * Part of invidious
+ * Copyright iv-org
+ * Licensed under AGPLv3
+*/
+
+
+/*
+ * login/Register pages
+*/
+
+.login-container,
+.register-container {
+    width: max-content;
+    margin: 12vh auto;
+    text-align: end;
+    padding: 2em;
+    border: 1px solid;
+}
+
+.login-container label,
+.register-container label {
+    width: max-content !important;
+    margin-right: 1.5em !important;
+}
+
+.login-container .login-submit-button,
+.register-container .register-submit-button {
+    width: max-content;
+    margin: 1.75em auto 1em auto;
+}
+
+.login-container p,
+.register-container p {
+    margin: 0 auto;
+    text-align: center;
+}
+
+.login-container .username-pass,
+.register-container .username-pass {
+	width: max-content;
+	margin: auto;
+	padding: 0 2em;
+}
+
+.captcha {
+	margin: 2em 0.5em;
+	text-align: center;
+	padding: 1em;
+}
+
+/* Background color accent using transparency */
+/* TODO: handle themes better, ffs */
+.light-theme .captcha { background-color: #0002; }
+.dark-theme  .captcha { background-color: #fff2; }
+
+@media (prefers-color-scheme: light) { .no-theme .captcha { background-color: #0002; } }
+@media (prefers-color-scheme: dark)  { .no-theme .captcha { background-color: #fff2; } }
```

Comment:
Are these colors missing 2 digit of the hex code?

Reply:
I'm not sure how theming is handled in invidious but this seems redundent

Reply:
> Are these colors missing 2 digit of the hex code?

No, it's the short RGBA notation. It's the same as doing `#FFFFFF22` (RRGGBBAA).

> I'm not sure how theming is handled in invidious but this seems redundent

Short answer: badly, lol
Long answer: This due to 1) not using JS for the "auto" theme (we only use CSS for light/dark mode detection) and 2) because we're not using `--var: <properties>;` and `<property>: var(--var);` in order to have IE11 compatibility.